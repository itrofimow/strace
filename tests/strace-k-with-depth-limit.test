#!/bin/sh
#
# Check strace -k --stacktrace-frames-limit for attached tracees.
#
# Copyright (c) 2020-2023 The strace developers.
# All rights reserved.
#
# SPDX-License-Identifier: GPL-2.0-or-later

KOPT_SHORT=-k
KOPT_LONG=--stack-traces
KOPT_EXTRA="--stacktrace-frames-limit 3"

. "${srcdir=.}"/strace-k.test
