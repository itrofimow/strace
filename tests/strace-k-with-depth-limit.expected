^chdir .*(__kernel_vsyscaln )?(__)?chdir f3 f2 _too many stack frames_
^SIGURG .*(__kernel_vsyscaln )?(__)?kill f3 f2 _too many stack frames_
